variables.container.tf

```diff
@@ -18,7 +19,8 @@ variable "blob_properties" {
       max_age_in_seconds = number
     })))
     delete_retention_policy = optional(object({
```

Comment:
Is there a reason we don't support the `permanent_delete_enabled` property? https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account#permanent_delete_enabled-1